engine/src/main/java/org/hibernate/search/engine/metadata/impl/AnnotationMetadataProvider.java

```diff
@@ -1772,6 +1753,68 @@ else if ( log.isTraceEnabled() ) {
 		}
 	}
 
+	private void logWarnings(XProperty member, IndexedEmbedded indexedEmbeddedAnnotation) {
+		if ( !isDepthNotSet( indexedEmbeddedAnnotation ) ) {
+			log.indexEmbeddedValueIgnored( member.getName(), "depth" );
+		}
+		if ( void.class != indexedEmbeddedAnnotation.targetElement() ) {
+			log.indexEmbeddedValueIgnored( member.getName(), "targetElement" );
+		}
+		if ( !isDefaultPrefix( indexedEmbeddedAnnotation ) ) {
+			log.indexEmbeddedValueIgnored( member.getName(), "prefix" );
```

Comment:
I'm not sure. Just wondering: we have the @Field(name) and the prefix, so we _could_ opt to use it.

If we don't currently then +1 to not change this suddendly and have a nice warning.